tests/game/invisibility-potion.test.ts

```diff
@@ -0,0 +1,195 @@
+		testGame(
+			{
+				playerOneDeck: [EthosLabCommon, InvisibilityPotion],
+				playerTwoDeck: [EthosLabCommon],
+				saga: function* (game) {
+					yield* playCardFromHand(game, EthosLabCommon, 'hermit', 0)
+					yield* playCardFromHand(game, InvisibilityPotion, 'single_use')
+					yield* applyEffect(game)
+					yield* endTurn(game)
+
+					yield* playCardFromHand(game, EthosLabCommon, 'hermit', 0)
+					yield* attack(game, 'secondary')
+
+					// Invisibility potion should block damage.
+					expect(
+						game.components.find(
+							RowComponent,
+							query.row.active,
+							query.row.opponentPlayer,
+						)?.health,
+					).toBe(EthosLabCommon.health)
+				},
+			},
+			{startWithAllCards: true, noItemRequirements: true, forceCoinFlip: true},
+		)
+	})
+	test('Invisibility Potion doubles damage on tails.', () => {
+		testGame(
+			{
+				playerOneDeck: [EthosLabCommon, InvisibilityPotion],
+				playerTwoDeck: [EthosLabCommon, BadOmen],
+				saga: function* (game) {
+					yield* playCardFromHand(game, EthosLabCommon, 'hermit', 0)
+					yield* endTurn(game)
+
+					yield* playCardFromHand(game, EthosLabCommon, 'hermit', 0)
+					yield* playCardFromHand(game, BadOmen, 'single_use')
+					yield* applyEffect(game)
+					yield* endTurn(game)
+
+					yield* playCardFromHand(game, InvisibilityPotion, 'single_use')
+					yield* applyEffect(game)
+					yield* endTurn(game)
+
+					yield* attack(game, 'secondary')
+
+					// Invisibility potion should double incoming damage.
+					expect(
+						game.components.find(
+							RowComponent,
+							query.row.active,
+							query.row.opponentPlayer,
+						)?.health,
+					).toBe(EthosLabCommon.health - EthosLabCommon.secondary.damage * 2)
+				},
+			},
+			{startWithAllCards: true, noItemRequirements: true, forceCoinFlip: true},
+		)
+	})
+	test('Invisibility Potion blocks Skizzleman Rare damage on heads.', () => {
+		testGame(
+			{
+				playerOneDeck: [EthosLabCommon, EthosLabCommon, InvisibilityPotion],
+				playerTwoDeck: [SkizzlemanRare, Anvil],
+				saga: function* (game) {
+					yield* playCardFromHand(game, EthosLabCommon, 'hermit', 0)
+					yield* playCardFromHand(game, EthosLabCommon, 'hermit', 1)
+					yield* playCardFromHand(game, InvisibilityPotion, 'single_use')
+					yield* applyEffect(game)
+					yield* endTurn(game)
+
+					yield* playCardFromHand(game, SkizzlemanRare, 'hermit', 0)
+					yield* playCardFromHand(game, Anvil, 'single_use')
+					yield* attack(game, 'secondary')
+
+					// Invisibility potion should block both of Skizzlman's attacks.
+					expect(
+						game.components.find(
+							RowComponent,
+							query.row.index(0),
+							query.row.opponentPlayer,
+						)?.health,
+					).toBe(EthosLabCommon.health - 30 /* Anvil */)
+					expect(
+						game.components.find(
+							RowComponent,
+							query.row.index(1),
+							query.row.opponentPlayer,
+						)?.health,
+					).toBe(EthosLabCommon.health - 10 /* Anvil */)
```

Comment:
Modify this test to check AFK Etho's health after playerTwo ends their turn
```suggestion
					yield* endTurn(game)
					expect(
						game.components.find(
							RowComponent,
							query.row.index(1),
							query.row.currentPlayer,
						)?.health,
					).toBe(EthosLabCommon.health - 10 /* Anvil only */)
```